c2cgeoportal/scaffolds/create/deploy/hooks/pre-restore-database.mako

```diff
@@ -11,23 +11,26 @@ DATABASES=$@
 # databases / tables restore.
 sudo apache2ctl stop
 
+if psql -lqt | cut -d \| -f 1 | grep -qw $DATABASES; then
```

Comment:
no `sudo -u postgresql`?
long parameters =>
```
psql --list --quiet --tuples-only | cut --delimiter='|' --fields=1 | grep --quiet --word-regexp $DATABASES
```

why `psql --quiet`?
why `grep --quiet`?
`--word-regexp` should be `--fixed-strings`...